CMakeLists.txt

```diff
@@ -396,6 +396,32 @@ install(TARGETS KVEngine Launcher
 install(FILES ${CMAKE_BINARY_DIR}/metadata.json
         DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/${EXEC_DIR}/)
 
+IF(WIN32)
+    install(CODE "
```

Comment:
Can you just add rules for every used library? It will be tedious at first but will remove metacode and simplify build.